components/applicationsets/kustomization.yaml

```diff
@@ -5,4 +5,6 @@ kind: Kustomization
 resources:
   - core-components-appset.yaml
   # - clusters-appset.yaml
-  #- worker-nodes-appset.yaml
+  # - worker-nodes-appset.yaml
+  - ztp-clusters-appset.yaml
```

Comment:
Should this be uncommented?

Reply:
just testing something will uncomment it

Reply:
Reverted the changes again. Please review let me know any further changes required